ext/hal/st/stm32cube/stm32f4xx/drivers/include/stm32f4xx_hal_conf.h

```diff
@@ -71,7 +71,6 @@
 #define HAL_SDRAM_MODULE_ENABLED
 #define HAL_HASH_MODULE_ENABLED
 #define HAL_GPIO_MODULE_ENABLED
-#define HAL_I2C_MODULE_ENABLED

```

Comment:
No other STM32 platform enables the i2c LL and the HAL. When it happens, the multiple definitions of  I2C_SPEED_STANDARD and I2X_SPEED_FAST occur.